### DISCUSSION TITLE
Gold pool

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#550] [Q&A] [answered]
A little while ago, a change was introduced to the MP books. According to the comment left there during that time it was done due to the gold pool being drained to quickly.

A long while ago during slaughterfest it was also mentioned that the gold used to buy items from stores or other in-game vendors would be put into some sort of gold pool.

The post stated that ''a percentage'' (https://trello.com/c/KExyPMMc) of that amount was added to the gold pool. I figured this would work out just fine. However, seeing that the gold pool is already nearing exhaustion or being drained too quickly I got a bit curious to what the percentage actually is. I was also curious for how long this pool had been accumulating gold? Has this been kept track of since the introduction of gold or was it only done so shortly after being added?

As for the issue with the gold pool, I believe the largest part of it lies within the gold pass rewards. Premium members are able to claim over 1000+ gold, which is more then the pass costed. The same goes for non pass users, which gain around 450 gold. To combat such a thing I believe it would be better to decrease the total yield of gold from pass completion or use a percentage of the pass cost. For example 50% (still quite high), which would be 500 gold. Gold rewards from post rewards seems to be fine, though I don't have the exact statistic on the entire economy on it. I would expect however that the rewards before post drained the most amount of gold.

I also have been thinking of a way to make the MPbooks a bit more engaging rather then just wait for timer to expire and move. The reason so much were accumulated within a short time seemed to be because they are extremely easy to obtain, you partically don't have to do anything. Joining the server is enough to obtain one. I'll create a post about that soon, but it involves the use of a seperate menu and new consumable(s) (which would be optional additions). The idea behind it, if it stays the same that is, is that the player can do small tasks which steeply increase the higher they get into the pass. Details on it follow later.

**Comment by MXKhronos** [comment, marked as answer]
Oh, I should had mentioned that the gold pool is not based on gold shop purchases and the trello post is outdated after various tweaks to how gold pools works.

Each event pass has their own gold pool and are sourced from purchasing the pass and purchasing levels, mostly level purchases. There is no longer a set percentage of gold sinked anymore, every single gold gets added to the pool. So gold spent on the event pass are still in circulation until the end of the event.

Based on last event pass's statistics, I increased the potential of obtaining gold, but with the addition of `mpbooks`, it was depleting the pool a bit quick initially. After the book drop rate tweaks, the statistics are at a stable amount of gold reserve to gold drop ratio currently at about 45% of pool gold are dropped, but this is not final so until the event is over it might be different.
